### PR TITLE
traffic tiles docs

### PR DESCRIPTION
**Pull request by tcql**
Rough draft on Mapbox traffic tiles docs

traff-dco

cc @aaronlidman

### CODE REVIEW

**Comment by aaronlidman** [comment]
@ajashton - Mind giving a quick review? Anything more that would be useful here?

**Comment by ajashton** [comment]
:+1: looks great to me. Not noticing anything obviously missing or unanswered.

**Comment by aaronlidman** [comment]
Thanks AJ

File: _posts/services/0001-10-21-mapbox-traffic-v1.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+### Line offsets
+
+Mapbox Traffic can be used to display congestion for both directions on two way roads. When styling congestion, it's recommended that you add a positive `line-offset` to the layer to visually separate the directions of travel.
+
+### Data updates
+
+Mapbox Traffic receives two different types of data update:
+
+- Traffic speeds and densities, used to derive `congestion` are updated approximately every 5 minutes
```
Comment (tmcw) [inline _posts/services/0001-10-21-mapbox-traffic-v1.md]: Grammar note:

Traffic speeds and densities, used to derive `congestion` are updated approximately every 5 minutes

^ this comma placement doesn't quite work. You could do

> Traffic speeds and densities, used to derive `congestion`, are updated approximately every 5 minutes

This attaches 'used to derive `congestion`' to 'Traffic speeds and densities' as an interjection. You could also drop the comma entirely. In this case there should be either two commas or none at all.

Comment (tmcw) [inline _posts/services/0001-10-21-mapbox-traffic-v1.md]: Seeing how this is used in context, you could also write:

- Traffic speeds and densities: used to derive `congestion`, updated approximately every 5 minutes
- Road geometries: based on OpenStreetMap, periodically updated

File: _posts/services/0001-10-21-mapbox-traffic-v1.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+When you publicly use styles or software that use Mapbox Traffic vector tiles, you must [display proper attribution](https://www.mapbox.com/help/attribution/).
+
+### Line offsets
+
+Mapbox Traffic can be used to display congestion for both directions on two way roads. When styling congestion, it's recommended that you add a positive `line-offset` to the layer to visually separate the directions of travel.
+
+### Data updates
+
+Mapbox Traffic receives two different types of data update:
```
Comment (tmcw) [inline _posts/services/0001-10-21-mapbox-traffic-v1.md]: Mapbox Traffic receives two different types of data updates:

**Comment by aaronlidman** [comment]
Thanks Tom.

**Pull request merged**